ios/FluentUI/Core/Theme/Tokens/TokenizedControl.swift

```diff
@@ -19,9 +19,20 @@ public protocol TokenizedControl {
 
 /// Internal extension to `TokenizedControl` that adds the ability to modify the active tokens.
 protocol TokenizedControlInternal: TokenizedControl {
-    /// Sets the current tokens to be used for drawing.
-    func updateCurrentTokens(_ tokens: TokenType)
+    /// Default token set.
+    var defaultTokens: TokenType { get }
 
     /// Fetches the current token override.
     var overrideTokens: TokenType? { get }
 }
+
+// MARK: - Extensions
+
+extension TokenizedControlInternal {
+    /// Returns the correct token set for a given tokenizable control.
+    func tokens(for fluentTheme: FluentTheme) -> TokenType {
```

Comment:
Could we move the fluentTheme in to TokenizedControlInternal, rather than passing it in? Seems a bit odd to say "get me the tokens for this theme, please" and it replies "hey, I saw you have override tokens, have those instead".

Reply:
Great idea! Yes, I can do that. I'll also rename the function to `resolvedTokens()` to keep it from just being `tokens()`, which would be pretty confusing.

Backstory: I was originally focused on getting the `@Environment` variable stored into this extension, which is not allowed. It didn't occur to me that I could just put the d